src/mumble/ServerHandler.cpp

```diff
@@ -800,6 +800,10 @@ void ServerHandler::serverConnectionConnected() {
 	mpa.set_opus(true);
 	sendMessage(mpa);
 
+	if (Global::get().s.bUdpDisabled) {
+		Global::get().sh->disableUdpMode(true);
```

Comment:
```suggestion
		disableUdpMode(true);
```
This should be applied to the current instance